src/main/scala/co/teapot/tempest/algorithm/MonteCarloPPR.scala

```diff
@@ -58,17 +73,36 @@ object MonteCarloPPR {
     }
   }
 
-  def estimatePPRParallel(graph: DirectedGraph, startIds: IndexedSeq[Int], params:
-  MonteCarloPageRankParams, threadCount: Int):
+  def estimatePPRParallel(graph: DirectedGraph,
+                          startIds: IndexedSeq[Int],
+                          firstStepDirection: EdgeDir,
+                          lengthConstraint: LengthConstraint,
+                          params: MonteCarloPageRankParams,
+                          threadCount: Int):
   collection.Map[Int, Double] = {
     params.setNumSteps(params.getNumSteps / threadCount)
+    params.setMinReportedVisits(params.getMinReportedVisits / threadCount)
```

Comment:
Good catch.  For example, if the client only wants nodes included if they were visited on 10 walks (to filter out noise), and we have 2 threads, my heuristic was that each thread must visit the node 10 / 2 times for the node to be included.  I realize now that isn't the case (e.g. it could be 4 visits on one thread, and 6 on the other).  I'll just remove that, and do the filtering after summing the individual results.